src/v2/ion/uiSchemaLabelTransformer.js

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { _, loc, $ } from 'okta';
+import Bundles from 'util/Bundles';
+import Logger from 'util/Logger';
+
+const SECURITY_QUESTION_PREFIX = 'enroll-authenticator.security_question.credentials.questionKey.';
+
+const I18N_OVERRIDE_MAPPINGS = {
```

Comment:
can you also put a comment that this would be used only till we have i18n keys in API itself..